<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MainWidgetTitle" name="widget_title" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="MainWidgetBtnGroup" name="widget_btnGroup" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_body">
          <widget class="MonitorMainWidget" name="page_monitorWidgwt"/>
          <widget class="VideoMainWidget" name="page_videoWidget"/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWidgetTitle</class>
   <extends>QWidget</extends>
   <header>MainWidgetTitle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWidgetBtnGroup</class>
   <extends>QWidget</extends>
   <header>MainWidgetBtnGroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MonitorMainWidget</class>
   <extends>QWidget</extends>
   <header>MonitorMainWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoMainWidget</class>
   <extends>QWidget</extends>
   <header>VideoMainWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
